src/universal/components/LinkNewTab/LinkNewTab.js

```diff
@@ -0,0 +1,22 @@
+import React, {PropTypes} from 'react';
+
+const LinkNewTab = (props) => {
+  const { children, href, title } = props;
```

Comment:
@ndey96 I removed the URL-helping here. It didn't sit well with me web-standards wise, and also, GitHub doesn't do any helping so I don't think we should either.

Let's use this is a starting point and if we need to grow something more ornate because our users are struggling, we will.